<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>w_Home</class>
 <widget class="QWidget" name="w_Home">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>795</width>
    <height>597</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>青岛艾普智能仪器TCP服务工具--by赵楠林</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QTableWidget" name="tableWidget">
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <attribute name="horizontalHeaderHighlightSections">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderHighlightSections">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>接收数据</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditData"/>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>发送数据</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditSend"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>发送</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonStart">
     <property name="text">
      <string>开始监听</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
